HW_18/task_1.py

```diff
@@ -0,0 +1,20 @@
+import re
+class Mail:
+    def __init__(self, email: str):
+        self.email = email
+        self.validate()
+
+
+    def validate(self):
+        current_mail = r'^[a-zA-Z0-9_.+-]+[^_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$'
```

Comment:
I don't get this part: `[a-zA-Z0-9_.+-]+[^_.+-]`. Are you trying to add two groups of symbols? Also, don't forget to escape symbols with special meaning (like dots, for example).